app/tests/test_inventory.py

```diff
@@ -0,0 +1,79 @@
+from unittest import TestCase
+from unittest.mock import patch
+
+from mock.mock_loader import mock
+from resources import inventory
+from schemes import item_not_found, success
+
+
+class TestInventory(TestCase):
+    def setUp(self):
+        mock.reset_mocks()
+
+        self.query_inventory = mock.MagicMock()
+        mock.wrapper.session.query.side_effect = {inventory.Inventory: self.query_inventory}.__getitem__
+
+    def test_list_inventory(self):
+        elements = [mock.MagicMock() for _ in range(5)]
+        self.query_inventory.filter_by.return_value = elements
+        self.assertEqual({"elements": [e.serialize for e in elements]}, inventory.list_inventory({}, "the-user"))
+        self.query_inventory.filter_by.assert_called_with(owner="the-user")
+
+    def test_exists_not_found(self):
```

Comment:
I think the prefix `test` is unnecessary -> Testing Guide

Reply:
> The individual tests are defined with methods whose names start with the letters `test`.

No, the prefix is necessary